apps/modernization-ui/src/suggestion/Suggestions.scss

```diff
@@ -4,4 +4,7 @@ ul.suggestions {
     border: 1px solid black;
     border-top: 0;
     padding: 0;
+    overflow: hidden;
+    max-height: 300px;
+    overflow-y: scroll;
```

Comment:
Suggestion: typically `overflow-y: auto` is better than `scroll` since `scroll` will always show the scroll bar even if it is not needed